src/index.html

```diff
@@ -6,9 +6,60 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Handjet:wght@200;400;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
```

Comment:
```suggestion
          alt="Moyo logo"
```
Checklist: "alt attribute should describe the image content"
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/afc3d150-7033-4c1c-ace1-72e64355950b)